### DISCUSSION TITLE
Plotting crystal orientation distributions

### DISCUSSION THREAD

**Discussion by J-AKAZ** [neperfepx/neper#122] [Q&A] [answered]
Hi Dears,
I have a question about Crystal orientation distribution. How can we plot orientation distribution from the .ori file as shown in the generation page? https://neper.info/generation.html#new-crystal-orientation-distribution
Thank you :)

**Comment by rquey** [comment, marked as answer]
The image on the Neper page was generated with one of my (crappy) in-house codes. It is not in Neper (yet), sorry...
